.changeset/next-link-read-only-2024-1-7-14-13-43.md

```diff
@@ -0,0 +1,5 @@
+---
+"@azure-tools/typespec-azure-core": patch
```

Comment:
```suggestion
"@azure-tools/typespec-azure-core": minor
```

Reply:
As I think this is not really a fix but a change

Reply:
Good point, thanks!